flang/include/flang/Parser/parse-tree.h

```diff
@@ -3738,6 +3738,33 @@ struct OmpOrderClause {
   std::tuple<std::optional<OmpOrderModifier>, Type> t;
 };
 
+// REF: [5.1:217-220], [5.2:293-294]
+//
+// init-clause -> INIT ([interop-modifier,] [interop-type,]
+//                              interop-type: interop-var)
```

Comment:
expand interop-modifier and interop-type as well in the comment section:
		interop-modifier: prefer_type(preference-list)
		interop-type: target, targetsync
			There can be at most only two interop-type.

Reply:
Addressed the comment on [#1c57d1e](https://github.com/swatheesh-mcw/llvm-project/pull/1/commits/1c57d1ea2b8e7095a834689ad5b832e6a2a84c37)